client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -194,4 +204,11 @@ export class CommentNavigation extends Component {
 	}
 }
 
-export default localize( UrlSearch( CommentNavigation ) );
+const mapDispatchToProps = {
+	recordChangeFilter: status => composeAnalytics(
+		recordTracksEvent( COMMENTS_STATS_GROUP + '_change_filter', { status } ),
```

Comment:
Not blocking but I usually like these as strings, so they're easy to search for. So for example:
```jsx
recordTracksEvent( 'calypso_comment_management_change_filter', { status } )
```